src/write_queue.rs

```diff
@@ -76,20 +76,19 @@ impl WriteQueue {
 
     pub async fn get_object(
         &self,
-        mut i: GetObjectInput,
+        i: GetObjectInput,
     ) -> Result<GetObjectOutput, GetObjectError> {
         let fname = self.to_file_name(i.bucket(), i.key());
         let stream = ByteStream::from_path(Path::new(&fname))
             .await
-            .map_err(|err| GetObjectError::NoSuchKey(NoSuchKey::builder().build()))?;
+            .map_err(|_| GetObjectError::NoSuchKey(NoSuchKey::builder().build()))?;
```

Comment:
Another option is to
Leave the name but prefix the unused name with underscore, like `_err` which just helps to
Preserve the original meaning a bit more.

Reply:
Hope I've addressed your concerns....